SWLOR.Game.Server/Feature/AbilityDefinition/Force/ForceSparkAbilityDefinition.cs

```diff
@@ -85,16 +93,25 @@ private void ForceSpark2()
             _builder.Create(FeatType.ForceSpark2, PerkType.ForceSpark)
                 .Name("Force Spark II")
                 .Level(2)
-                .HasRecastDelay(RecastGroup.ForceSpark, 20f)
-                .RequirementFP(2)
+                .HasRecastDelay(RecastGroup.ForceSpark, 6f)
+                .HasActivationDelay(2f)
                 .IsCastedAbility()
                 .HasMaxRange(10f)
                 .IsHostileAbility()
                 .UsesAnimation(Animation.LoopingConjure1)
                 .DisplaysVisualEffectWhenActivating()
                 .HasImpactAction((activator, target, level, location) =>
                 {
-                    Impact(activator, target, 14, 4, 2, Tier2Tag, 12);
+                    var willBonus = GetAbilityModifier(AbilityType.Willpower, activator);
```

Comment:
Similar to other spots - please wrap this functionality into a method and pass in parameters to it. The logic is the same across both tiers, so there's no reason to duplicate the code. This will improve maintainability as changes only need to occur in one place.